demo/addToast.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+showToast = (toastID) => {
```

Comment:
This should be located under `apiExamples/` and should be ran under a file that runs all javascript examples for each respective page, like so: https://github.com/AlaskaAirlines/auro-combobox/blob/main/demo/index.js

Reply:
@jordanjones243 please consult with @cynthiaricomendoza-alaskaair to assist with this update. This follows the new example.js model that I am not all that failure with.

Reply:
met with Jordan yesterday and pushed up changes for this!